keps/sig-scheduling/4328-affinity-based-eviction/kep.yaml

```diff
@@ -0,0 +1,44 @@
+title: "Affinity Based Eviction" 
+kep-number: 4328
+authors:
+  - "@AxeZhan"
+owning-sig: sig-scheduling
+participating-sigs:
+  - sig-scheduling
+  - sig-node
+status: provisional
+creation-date: 2023-12-13
+reviewers:
+  - "@alculquicondor"
+  - TBD
+approvers:
+  - "@alculquicondor"
+  - TBD
```

Comment:
Currently no, I'll bring this KEP to their slack channel and try to find one in the next following days :)

Reply:
did you make progress with this?

Reply:
https://github.com/kubernetes/enhancements/pull/4329#issuecomment-1865032661
Both @kannon92 and @ffromani will take on this role.

Reply:
But AFAIK, they are not sig-node approvers

Reply:
I'm indeed _not_ a sig-node approver.

Reply:
Same, but I don't really understand if a sig-node approver is required for this.

Reply:
> Same, but I don't really understand if a sig-node approver is required for this.

+1 for me. Do we also need a sig-node approver when this KEP is owned by sig-scheduling?
I thought we only need reviewers from sig-node before, sry.

Reply:
KEPs are to be approved by TLs to be more specific. But since sig-node is only a participating SIG, I would accept a review from an approver.

Then I can approve from SIG scheduling

Reply:
@AxeZhan can you add this KEP to the agenda for sig-node meeting next week? I'll bring it up (or if you can attend) and see if we can get an approver to review. I'll bring up @alculquicondor point.

Reply:
> @AxeZhan can you add this KEP to the agenda for sig-node meeting next week? I'll bring it up (or if you can attend) and see if we can get an approver to review. I'll bring up @alculquicondor point.

@kannon92 That would be very helpful! Thanks a lot! I added this to the agenda for Jan 23.  
However I don't think I can attend😢, the meeting time is not friendly to me(it will be 2am for me).

Reply:
Hi, I brought this up in sig-node KEP planning. @dchen1107 agreed to review from sig-node.